### ISSUE TITLE
Updating LCC for all Technologies

### ISSUE THREAD

**Issue by jillianderson8** [EMRG-SFU/cims#70]
Look into what happens with LCC for existing stock. 

We need to make sure the LCC for all technologies is being updated at each iteration, even for technologies which are not part of the market competition (e.g. those that are no longer available). This update needs to happen to account for changing fuel costs. 

Currently, I do not think we are updating these LCCs. A possible implication of this could be that we have some technologies whose LCC is never being calculated (e.g. pyCIMS.Canada.Alberta.Residential.Buildings.Ranges[NG existing]). 

I need to investigate this to see what is currently happening. Once that is done I can make adjustments to ensure the proper updates are occurring.
